docs/4-cloud-computing/4.3.7-app-service.md

```diff
@@ -0,0 +1,97 @@
+---
+docs/4-cloud-computing/4.3.7-app-service.md:
+  category: Cloud Computing
+  estReadingMinutes: 10
+  exercises:
+    -
+      name: Deploy Simple Web Application to App Service
+      description: Create A Web App and Scale and Monitor
+      estMinutes: 240
+      technologies:
+      - Azure
+      - Azure App Service
+      - Azure CLI
+---
+# Azure App Service
+
+Azure App Service is a fully managed platform for building, deploying, and scaling web apps. It supports various programming languages and frameworks like .NET, Java, Node.js, Python, and more, allowing developers to build web and mobile back-ends with ease. Through Azure App Service, you can integrate with various Azure tools like Azure DevOps for CI/CD and take advantage of built-in features such as custom domains and SSL certificates. It's designed with best practices in mind, providing scalability, robust monitoring, and security, thereby enabling a seamless and efficient app hosting solution.
+
+> [Azure App Service](https://learn.microsoft.com/en-us/azure/app-service/) Documentation
+
+### Exercise 1: Deploy a Java Application
+
+We will be using liatrio's version of the spring-petclinic for this exercise, its a simple java application that deploys a web app for a virtual pet clinic.
+
+1. Fork [Spring-Petclinic](https://github.com/liatrio/spring-petclinic.git). (We will be using this later for Continuous Deployment)
+
+2. Clone your new fork.
+
+```bash
+git clone https://github.com/YOUR_NAME/spring-petclinic.git
+```
+
+!> There is an official repo for the spring-petclinic, we will not be using that since it contains different build test that may not be useful for our purposes and might break the application on build.
```

Comment:
There is a grammar error here.

-  "we will not be using that since it contains different build test**s**"
-  "we will not be using that since it contains **a** different build test"